docs/jailer.md

```diff
@@ -116,9 +116,6 @@ as the path which contains them):
 - `/srv/jailer/firecracker/551e7604-e35c-42b3-b825-416853441234/root/firecracker`
   (copied from `/usr/bin/firecracker`)
 
-Firecracker will create its API socket at
```

Comment:
Firecracker still creates this API socket in this example, just the path is changed. We should keep this in the readme.

Reply:
I thought this line was in the middle of explaining the jailer's operation. so I moved the information about socket creation at the end for making it more clear 🙂.

Reply:
Ok.